symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue

```diff
@@ -0,0 +1,351 @@
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <div class="orangehrm-optional-field-row">
+              <oxd-text tag="p" class="orangehrm-optional-field-label">
+                Send Test Mail
+              </oxd-text>
+              <oxd-switch-input v-model="sendTestMailEditable" />
+            </div>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row v-if="sendTestMailEditable">
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Test Email Address"
+                v-model="emailConfiguration.testEmailAddress"
+                :rules="rules.testEmailAddress"
+                :disabled="!sendTestMailEditable"
+                required
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <required-text />
+          <submit-button/>
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@orangehrm/core/util/services/api.service';
+import SwitchInput from '@orangehrm/oxd/src/core/components/Input/SwitchInput';
+import { required, validEmailFormat } from '@orangehrm/core/util/validation/rules'
+import { shouldNotExceedCharLength } from '@/core/util/validation/rules'
+
+export default {
+  props: {
+    pathToSendmail: {
+      type: String,
+      required: true,
+    },
+  },
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/admin/email-configuration',
+    );
+    return {
+      http,
+    };
+  },
+
+  components: {
+    'oxd-switch-input': SwitchInput,
+  },
+
+  data() {
+    return {
+      userSecureConnection: false,
+      sendTestMailEditable: false,
+      isLoading: false,
+      emailConfiguration: {
+        mailType: '',
+        sentAs: '',
+        pathToSendmail: this.pathToSendmail,
+        smtpHost: '',
+        smtpPort: null,
+        smtpUsername: '',
+        smtpPassword: '',
+        smtpAuthType: '',
+        smtpSecurityType: '',
+        testEmailAddress: '',
+      },
+      rules: {
+        mailType: [required, shouldNotExceedCharLength(50)],
+        sentAs: [required, shouldNotExceedCharLength(250), validEmailFormat],
+        smtpHost: [required, shouldNotExceedCharLength(250)],
+        smtpPort: [shouldNotExceedCharLength(10)],
+        smtpUsername: [required, shouldNotExceedCharLength(250)],
+        smtpPassword: [required, shouldNotExceedCharLength(250)],
+        smtpAuthType: [shouldNotExceedCharLength(50)],
+        smtpSecurityType: [shouldNotExceedCharLength(50)],
+        testEmailAddress: [required, shouldNotExceedCharLength(250), validEmailFormat],
+      },
+    };
+  },
+
+  methods: {
+    onSave() {
+      this.isLoading = true;
+      this.http.http
+        .put('api/v2/admin/email-configuration', {
```

Comment:
use `this.http.request()` . check [here](https://github.com/orangehrm/orangehrm/blob/ef04d1a289e94bb8a624030a0daf473fad964f34/symfony/client/src/orangehrmPimPlugin/pages/employee/EmployeeJob.vue#L320) for reference.